portfolio/src/main/java/com/google/sps/servlets/RandomStringServlet.java

```diff
@@ -27,6 +27,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Send the JSON as the response
     response.setContentType("application/json;");
     response.getWriter().println(json);
+
```

Comment:
Adding the blank line was unnecessary?

You're probably thinking that this is nitpicking. The point is to get into a habit of having PRs that don't make any unnecessary changes.